Extension_CloudFlare_View_Dashboard.js

```diff
@@ -1,6 +1,6 @@
 jQuery(function() {
     jQuery('#w3tc_dashboard [type=submit]').bind('click', function(){
-        jQuery(this).attr('was_clicked','yes');
+        jQuery(this).prop('was_clicked','yes');
```

Comment:
@cssjoe how about that? i guess there are like 100 similar non-default props.

Reply:
I reverted those.